UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/SubscriptionServiceJob.cs

```diff
@@ -47,18 +47,41 @@ public async Task ProcessQueueMessage([QueueTrigger("%SubscriptionStorageConfigu
                 ExternalNotificationEntity externalNotificationEntity = new()
                 {
                     ResponseStatusCode = _d365CallbackConfiguration.Value.SucceededStatusCode,
-                    ResponseDetails = Convert.ToString(DateTime.UtcNow)
+                    ResponseDetails = "Successfully added subscription @Time: " + Convert.ToString(DateTime.UtcNow)
```

Comment:
This might be better as an interpolated string:
```suggestion
                    ResponseDetails = $"Successfully added subscription @Time: {DateTime.UtcNow}"
```

Reply:
Thanks